services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/component/DccRulesNotFetchedStructureProviderTest.java

```diff
@@ -166,11 +166,11 @@ void shouldContainEmptyInvalidationRules() throws FetchBusinessRulesException {
     assertEquals("ehn-dgc", digitalGreenCertificates.getName());
 
     List<Writable<WritableOnDisk>> businessRulesArchives = getBusinessRulesArchives(digitalGreenCertificates);
-    assertThat(businessRulesArchives).hasSize(4);
+    // json validation is only happening with an actual feign client, so we get all elements here, even
+    // elements that don't fit the json schema
+    assertThat(businessRulesArchives).hasSize(5);
 
     assertThat(businessRulesArchives.stream().filter(filterByArchiveName("onboarded-countries"))).hasSize(1);
-    // acceptance rules are invalid, they do not pass validation schema, thus archive won't be overwritten.
-    assertThat(businessRulesArchives.stream().filter(filterByArchiveName("acceptance-rules"))).isEmpty();
```

Comment:
do we have anywhere else a test, which ensures: that an archive is NOT overwritten, in case it would be empty?

Reply:
Can we have a quick discussion about this?